src/_includes/partials/components/filters.njk

```diff
@@ -0,0 +1,83 @@
+{% endfor %}
+<div id="filters" class="filters flow">
+    <h3>{% __ 'filters' %}</h3>
+    <div class="flow">
+        <fieldset>
+            <legend class="visually-hidden">{% __ 'topic' %}</legend>
+            {% disclosure 'topic' %}
+            <ul id="filter-topic">
+                {% for topic in collections['topics_' + lang] %}
+                    {% if topic.data.uuid in filterableOptions.topics %}
+                    <li>
+                        <input type="checkbox" id="{{ topic.data.uuid }}" name="topic" value="{{ topic.data.uuid }}" data-filter-key="topics" />
+                        <label for="{{ topic.data.uuid }}">{{ topic.data.title }}</label>
+                    </li>
+                    {% endif %}
+                {% endfor %}
+            </ul>
+            {% enddisclosure %}
+        </fieldset>
+        <fieldset>
+            <legend class="visually-hidden">{% __ 'type' %}</legend>
+            {% disclosure 'type' %}
+            <ul id="filter-type">
+                {% for value in filterableOptions.types %}
+                <li>
+                    <input type="checkbox" id="{{ value }}" name="type" value="{{ value }}" data-filter-key="type" />
+                    <label for="{{ value }}">{% __ value %}</label>
+                </li>
+                {% endfor %}
+            </ul>
+            {% enddisclosure %}
+        </fieldset>
+        <fieldset>
+            <legend class="visually-hidden">{% __ 'project' %}</legend>
+            {% disclosure 'project' %}
+            <ul id="filter-project">
+                {% for project in collections['projects_' + lang] %}
+                    {% if project.data.uuid in filterableOptions.projects %}
+                    <li>
+                        <input type="checkbox" id="{{ project.data.uuid }}" name="project" value="{{ project.data.uuid }}" data-filter-key="projects" />
+                        <label for="{{ project.data.uuid }}">{{ project.data.title }}</label>
+                    </li>
+                    {% endif %}
+                {% endfor %}
+            </ul>
+            {% enddisclosure %}
+        </fieldset>
+        <fieldset>
+            <legend class="visually-hidden">{% __ 'language' %}</legend>
+            {% disclosure 'language' %}
+            <ul id="filter-language">
+                <li>
+                    <input type="checkbox" id="language-en" name="language" value="en" data-filter-key="language" />
+                    <label for="language-en">{{ supportedLanguages["en"].name }}</label>
+                </li>
+                <li>
+                    <input type="checkbox" id="language-fr" name="language" value="fr" data-filter-key="language" />
+                    <label for="language-fr">{{ supportedLanguages["fr"].name }}</label>
+                </li>
+            </ul>
+            {% enddisclosure %}
+        </fieldset>
```

Comment:
We need to check for resources in French and hide these filters if there aren't any otherwise the behaviour is a bit confusing.